packages/app-frontend/src/features/components/ui/components/icons.ts

```diff
@@ -0,0 +1,34 @@
+const icons = require.context(
+  '@akryum/md-icons-svg/svg/',
+  true,
+  /materialicons\/24px\.svg$/,
+)
+
+export default {
+  install () {
+    const sprites = ['']
+    let spriteIndex = 0
+    // Load all the SVG symbols
+    icons.keys().forEach((key, index) => {
+      let result = icons(key)
```

Comment:
Hi!

This code is return a base64 encoded content with the string "module.exports = ....", I guess is normal because we are using require.context.

So, at the end is reporting errors all code at line 15 because the regular expresions will return a null (then cannot be destruct from null).

I tested with a code like:

```
      const base64Prefix = 'data:image/svg+xml;base64,'
      if (result.startsWith(base64Prefix)) {
        const positionPrefix = result.indexOf(base64Prefix) + base64Prefix.length
        const base64Content = result.substring(positionPrefix)
        result = atob(base64Content)
        const module: { exports?: string } = {}
        eval(result)
        result = module.exports
      }
```

before using the regular expression.